generator/subaru/_subaru_global.dbc

```diff
@@ -136,11 +136,11 @@ BO_ 290 ES_LKAS: 8 XXX
  SG_ LKAS_Output : 16|13@1- (-1,0) [-8191|8191] "" XXX
  SG_ LKAS_Request : 29|1@0+ (1,0) [0|1] "" XXX
 
-BO_ 292 ES_LKAS_ALT: 8 XXX
+BO_ 292 ES_LKAS_ANGLE: 8 XXX
  SG_ CHECKSUM : 0|8@1+ (1,0) [0|1] "" XXX
  SG_ COUNTER : 8|4@1+ (1,0) [0|1] "" XXX
  SG_ LKAS_Request : 12|1@1+ (1,0) [0|1] "" XXX
- SG_ LKAS_Output : 40|17@1- (-1,0) [0|1] "" XXX
+ SG_ LKAS_Output : 40|17@1- (-0.01,0) [0|1] "deg" XXX
```

Comment:
Can you rename this to indicate it's an angle too?

Reply:
And add a message comment describing which cars generally use this control message